providers/edu/harvarddataverse/harvester.py

```diff
@@ -1,5 +1,44 @@
-from share.harvest.harvester import Harvester
+from django.conf import settings
+
+from furl import furl
+
+from share import Harvester
 
 
 class HarvardDataverseHarvester(Harvester):
-    pass
+
+    def __init__(self, app_config):
+        super().__init__(app_config)
+        self.url = 'https://dataverse.harvard.edu/api/search/'
+        self.type = 'dataset'
+        self.MAX_ITEMS_PER_REQUEST = 1000
+
+    def do_harvest(self, start_date, end_date):
+        end_date = end_date.date()
+        start_date = start_date.date()
+
+        return self.fetch_records(furl(self.url).set(query_params={
+            'q': '*',
+            'type': self.type,
+            'per_page': self.MAX_ITEMS_PER_REQUEST,
+            'key': settings.DATAVERSE_API_KEY,
+            'sort': 'date',
+            'order': 'asc',
+            'fq': 'dateSort:[{}T00:00:00Z TO {}T00:00:00Z]'.format(start_date, end_date)
```

Comment:
Would you mind adding `.isoformat()` here? The default `__str__` of a datetime is different from `.isoformat()`

Technically it does not matter because they are dates not datetimes. It just removes any ambiguity.